### PR TITLE
add xeve/d and dovi_tool: install cargo-c

### PR DESCRIPTION
**Pull request by L4cache**
do_rustcinstall is called
https://github.com/m-ab-s/media-autobuild_suite/blob/90a968a0392dcf41a5eec0c6f1e6c5b33ee5ea27/build/media-suite_compile.sh#L1406

xeve/d building
closes: #2733

### CODE REVIEW

**Comment by L4cache** [comment]
How to choose different EVC profile though... maybe a dedicated option in .ini? But EVC is hardly interesting enough to make me do that.

File: build/media-suite_compile.sh [hunk 1]
```diff
@@ -1963,6 +1963,30 @@ if [[ $bits = 64bit && $vvdec = y ]] &&
...
+    do_checkIfExist
+fi
+
+_check=(bin-video/xevd_app.exe xevd/xevd{,_exports}.h libxevd.a xevd.pc)
+if [[ $ffmpeg != no ]] && enabled libxevd &&
+    do_vcs "$SOURCE_REPO_XEVD"; then
+    do_uninstall bin-video/libxevd.dll lib/libxevd.dll.a.dyn "${_check[@]}"
+    sed -i 's/-Werror //' CMakeLists.txt
+    do_cmakeinstall video
+    # no way to disable chared lib building in cmake
```
Comment (1480c1) [inline build/media-suite_compile.sh:1984]: misspelling

Comment (L4cache) [reply build/media-suite_compile.sh:1984]: lol

**Comment by 1480c1** [comment]
Thanks

**Pull request merged**